### PR TITLE
Add cluster autoscaler parameters for rosa

### PR DESCRIPTION
**Pull request by sunzhaohua2**
Support to enable cluster autoscaler while install clusters
Update a stable job to confirm this works well
@yasun1 PTAL, the job failure is not related to this pr

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-ec-aws-rosa-sts-f7

**Comment by sunzhaohua2** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-ec-aws-rosa-sts-f7

**Comment by sunzhaohua2** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-ec-aws-rosa-sts-f7

**Comment by sunzhaohua2** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-ec-aws-rosa-sts-f7

File: ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh [hunk 1]
```diff
@@ -256,13 +258,18 @@ if [[ "$ENABLE_AUTOSCALING" == "true" ]]; then
...
+  record_cluster "nodes" "min_replicas" ${MIN_REPLICAS}
+  record_cluster "nodes" "max_replicas" ${MAX_REPLICAS}
 else
   COMPUTE_NODES_SWITCH="--replicas ${REPLICAS}"
   record_cluster "nodes" "replicas" ${REPLICAS}
 fi
 
+CONFIGURE_CLUSTER_AUTOSCALER_SWITCH=""
+if [[ "$ENABLE_AUTOSCALING" == "true" ]] && [[ "$CONFIGURE_CLUSTER_AUTOSCALER" == "true" ]] && [[ "$HOSTED_CP" == "false" ]]; then
+  CONFIGURE_CLUSTER_AUTOSCALER_SWITCH="--autoscaler-balance-similar-node-groups --autoscaler-skip-nodes-with-local-storage --autoscaler-ignore-daemonsets-utilization --autoscaler-scale-down-enabled"
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:270]: Just a question, in the previous code, there are many options. If we do not set them, will they or their logic be used as default?

Comment (sunzhaohua2) [reply ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:270]: Yes, all these options have default values

File: ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -x
```
Comment (liangxia) [inline ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:6]: I think `set -x` generally used for debug/development.  Is it expected that it's still there given the steps are already verified ?

Comment (sunzhaohua2) [reply ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh]: Thanks, have removed

**Pull request merged**